controllers/cloud.redhat.com/providers/serviceaccount/default.go

```diff
@@ -88,7 +88,7 @@ func (sa *serviceaccountProvider) Provide(app *crd.ClowdApp, c *config.AppConfig
 
 		labeler := utils.GetCustomLabeler(nil, nn, app)
 
-		if err := CreateServiceAccount(sa.Cache, CoreDeploymentServiceAccount, sa.Env.Spec.Providers.PullSecrets, nn, labeler); err != nil {
+		if err := CreateServiceAccount(sa.Cache, CoreDeploymentServiceAccount, nn, labeler); err != nil {
```

Comment:
We're gonna have conflicts here depending on who gets merged first. Just a heads up

Reply:
Yeh